plugins/module_utils/storage/dell/utils.py

```diff
@@ -303,7 +301,7 @@ def find_compatible_powerscale_sdk(module_params, module=None):
             array_version = major + "_" + minor + "_0"
             if module == "ads":
                 # Adding a workaround for home_directory_template failure in later versions of SDK.
-                compatible_powerscale_sdk = "isilon_sdk.v9_1_0"
+                compatible_powerscale_sdk = "isilon_sdk.v9_5_0"
```

Comment:
we can remove the if condition, as both scenarios have the same action

Reply:
Done.